chall01/ptuukkan.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
+
+import sys
+
+codes = {
+	"a": ".-",
+	"b": "-...",
+	"c": "-.-.",
+	"d": "-..",
+	"e": ".",
+	"f": "..-.",
+	"g": "--.",
+	"h": "....",
+	"i": "..",
+	"j": ".---",
+	"k": "-.-",
+	"l": ".-..",
+	"m": "--",
+	"n": "-.",
+	"o": "---",
+	"p": ".--.",
+	"q": "--.-",
+	"r": ".-.",
+	"s": "...",
+	"t": "-",
+	"u": "..-",
+	"v": "...-",
+	"w": ".--",
+	"x": "-..-",
+	"y": "-.--",
+	"z": "--..",
+	" ": " "
+}
```

Comment:
using dictionary - smart! 
idk if you hardcoded it but if you did, you could compose it automatically in python like that: 
```
morse_string = '.- -... -.-. -.. . ..-. --. .... .. .--- -.- .-.. -- -. --- .--. --.- .-. ... - ..- ...- .-- -..- -.-- --..'
morse_list = morse_string.split(" ") # '\n' if you take one from the assignment
codes = dict(zip(string.ascii_lowercase, morse_list))
```